LAPACKE/CMakeLists.txt

```diff
@@ -58,7 +55,7 @@ else (USE_XBLAS)
 endif(USE_XBLAS)
 
 lapack_install_library(lapacke)
-INSTALL( FILES ${LAPACKE_INCLUDE} DESTINATION include )
+INSTALL( FILES ${LAPACKE_INCLUDE} ${LAPACK_BINARY_DIR}/include/lapacke_mangling.h DESTINATION include )
```

Comment:
I had missed adding  ${LAPACK_BINARY_DIR}/include/lapacke_mangling.h to the installed files in the earlier commit.  

This was a problem when using default cmake "find_package(LAPACKE REQUIRED)" from an external package.